.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: Test urdf_geometry_parser
+on:
+  pull_request:
+  push:
+    branches:
+      - kinetic-devel
+  schedule:
+    # Run every morning to detect flakiness and broken dependencies
+    - cron: '17 8 * * *'
+
+jobs:
+  industrial_ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: kinetic, ROS_REPO: main}
+          - {ROS_DISTRO: kinetic, ROS_REPO: testing}
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
+          - {ROS_DISTRO: noetic, ROS_REPO: main}
+          - {ROS_DISTRO: noetic, ROS_REPO: testing}
```

Comment:
I think this could actually be simplified to something along the lines of:
```
 - ROS_DISTRO: {kinetic, melodic, noetic}
 - ROS_REPO: {main, testing}
```

But I think it's clearer as it is, and also matches our other repos better.